packages/cli/src/sso/saml/views/saml-connection-test-failed.ts

```diff
@@ -1,10 +1,8 @@
+import { compile } from 'handlebars';
+
 import type { SamlUserAttributes } from '../types/saml-user-attributes';
 
-export function getSamlConnectionTestFailedView(
-	message: string,
-	attributes?: SamlUserAttributes,
-): string {
-	return `
+const failedTemplate = compile<{ message?: string; attributes?: SamlUserAttributes }>(`
     <http>
```

Comment:
not only do we have inline handlebars + template strings 🤦🏽 , looks like we can't even write valid markup. All of this code needs more time and attention.
```suggestion
    <html>
```